test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -805,6 +807,12 @@ def init_moms(cls, init_mom_func=None, skip='nomom'):
                                       func=init_mom_func)
         if cls.moms:
             cls.mom = cls.moms.values()[0]
+
+        for mom in cls.moms.values():
+            if not mom.isUp():
+                cls.logger.error('mom ' + mom.hostname + ' is down')
+                mom.pi.start_mom(mom.hostname)
```

Comment:
Why are we starting mom?  Also, if mom is down, mom.isUp() will wait like 30s for the mom to come up before returning.   You might want to give it a max_attempts